zebra/interface.c

```diff
@@ -805,21 +805,26 @@ void if_nbr_ipv6ll_to_ipv4ll_neigh_update(struct interface *ifp,
 	char buf[16] = "169.254.0.1";
 	struct in_addr ipv4_ll;
 	char mac[6];
+	ns_id_t ns_id;
 
 	inet_pton(AF_INET, buf, &ipv4_ll);
 
 	ipv6_ll_address_to_mac(address, (u_char *)mac);
 
+	if (!vrf_is_backend_netns())
```

Comment:
Shouldn't this just be a call to a accessor function?

zvrf = vrf_info_lookup(ifp->vrf_id);
zns = zvrf->zns;

This way we don't care about the backend.  Just get the zns